deploy.sh

```diff
@@ -738,6 +738,11 @@ function main {
   # Only use sudo if not running as root
   [ "$(id -u)" -ne 0 ] && SUDO=sudo || SUDO=""
 
+  if [[ $(uname -m) != "x86_64" ]]; then
+    warn "Only 64-bit machines are supported"
+    error "Unsupported machine: $(uname -m)"
+    exit 1
```

Comment:
Error do not need exit right? you can send the error code as second argument to the error function.